README.md

```diff
@@ -76,25 +65,68 @@ If the user agent doesn't support installation, present a simple link:
+The user agent can transform the element into a simple 'Launch'-style button, a highly requested feature from web developers. When clicked, it should follow existing launch protocols.
 
 <img alt='A button whose text reads "Launch YouTube Music, from music.youtube.com", with an icon signifying the action of launching.' src='./launch-simple.png' width=200>
 
-## Error handling
+User agents must take care with side channels such as width, as developers must not be able to detect the change in the element's content to avoid fingerprinting concerns.
+
+## Error handling / debuggability
 
 The element offers event-driven hooks allowing developers to understand users' interactions, reusing [`InPagePermissionMixin`][mixin] concepts like `promptaction`, `promptdismiss`, and `validationstatuschange` events, `isValid` and `invalidReason` attributes, etc. Additional events will be needed for failures related to manifest fetching/parsing.
 
 Validation errors could include violations of the generally applicable [presentation restrictions][security] for permission elements, as well as data validation errors when processing the referenced manifest.
 
 That said, developers wouldn't actually need to hook into any of those attributes for the simplest
-cases: `<install installurl="https://example.com/"></install>` would be sufficient for 
-straightforward use cases of offering installation.
+cases: `<install></install>` and `<install installurl="..."></install>` would be sufficient for straightforward use cases of offering installation.
 
 [pepc]: https://github.com/WICG/PEPC/
 [geolocation]: https://github.com/WICG/PEPC/blob/main/geolocation_explainer.md
 [mixin]: https://wicg.github.io/PEPC/permission-elements.html#permission-mixin
 [security]: https://github.com/WICG/PEPC/blob/main/explainer.md#security-abuse
 
+Design considerations
+--------------
+
+### Install by manifesturl?
+
+Should the `installurl` attribute be supplemented with, or even replaced by, a `manifesturl` attribute? e.g. https://youtube.com/manifest.webmanifest
+
+Issue - [Specifying manifests vs documents](https://github.com/WICG/install-element/issues/5#issuecomment-3613279651)
+
+Functionally, installing from a manifest file directly avoids the overhead of loading the document at `installurl` in the background, and makes it more feasible to render custom information in the button. It also reduces complexity around manifest id requirements, as the manifest file is the source of truth for an app's identity. However, it introduces a variety of additional concerns -
+
+* **Preventing manifest spoofing** - Manifests are not required to be served from the same origin as the site (e.g. CDN). To avoid spoofing, we'd need to load `start_url` in the background, and verify its manifest matches the provided attribute value. Alternatively, we could require that manifests be on the same origin.
```

Comment:
Another idea to save for later: could we optimistically assume the manifest is correct, present what it is saying, then fail the install later when we do load the site if it doesn't point back to the provided manifest url? 

No changes needed in this PR. Just an idea to explore.